runtime/near-wallet-contract/src/lib.rs

```diff
@@ -0,0 +1,32 @@
+#![doc = include_str!("../README.md")]
+
+use near_vm_runner::ContractCode;
+use once_cell::sync::OnceCell;
+use std::path::Path;
+
+/// Temporary (placeholder) Wallet Contract. Read from file once, then cache in memory.
+pub fn wallet_contract() -> &'static ContractCode {
+    static CONTRACT: OnceCell<ContractCode> = OnceCell::new();
+    let file_name = if cfg!(feature = "nightly") {
+        "nightly_wallet_contract.wasm"
+    } else {
+        "wallet_contract.wasm"
+    };
```

Comment:
What I would reccomed us to do is to discuss how upgrades will work and make sure that it's going to be possible to add this support painlessly in the future but without implemeting full support for it from day one. 

If we need to add some sort of versioning let's do it from day one. It could be some property of the blockchain at the time the eth implicit account was created, stored in the account itself. It can be the hash of the contract, the protocol version, some new dedicated version just for the wallet contract or something else.

Reply:
> Just for my full understanding, why do you need support for nightly here?
> 
> I'm fully onboard being future proof and supporting upgrading the contract but unless you need it now I would keep it simple and skip that part. I'm happy to be convinced otherwise though :)

I removed support for nightly. It not used currently and it is easy to add.



> What I would reccomed us to do is to discuss how upgrades will work and make sure that it's going to be possible to add this support painlessly in the future but without implemeting full support for it from day one.
> 
> If we need to add some sort of versioning let's do it from day one. It could be some property of the blockchain at the time the eth implicit account was created, stored in the account itself. It can be the hash of the contract, the protocol version, some new dedicated version just for the wallet contract or something else.

Will discuss it at the today's meeting.